templates/test/cluster-template-prow-nvidia-gpu.yaml

```diff
@@ -175,10 +175,10 @@ spec:
       osDisk:
         diskSizeGB: 128
         managedDisk:
-          storageAccountType: ${AZURE_GPU_NODE_STORAGE_TYPE:=Standard_LRS}
+          storageAccountType: Premium_LRS
         osType: Linux
       sshPublicKey: ${AZURE_SSH_PUBLIC_KEY_B64:=""}
-      vmSize: ${AZURE_GPU_NODE_MACHINE_TYPE:=Standard_NV6}
+      vmSize: ${AZURE_NODE_MACHINE_TYPE}
```

Comment:
This seems unrelated updating to 1.16. Was this needed or is related to another change set?